src/test/java/catalinazoldi/teorie/selenium2/JavaExceptions.java

```diff
@@ -0,0 +1,44 @@
+package catalinazoldi.teorie.selenium2;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+          public class JavaExceptions {
+              public static void main(String[] args) throws IOException {
+                  //checkedExceptions();
+                  //uncheckedException();
+                  //System.out.println("Am executat functia main ");
+              }
+
+              //checked exceptions
+              static void checkedExceptions() throws IOException {
+                  FileReader fileReader = new FileReader((System.getProperty(user.dir")+"\src\test\java\catalinazoldi\teorie\selenium2\text.txt")
```

Comment:
Here you have some compilation errors. Please fix them as in the screenshot -> 
![image](https://user-images.githubusercontent.com/18215854/215511407-dea58a3a-bf1b-4581-9c52-8bee5658a002.png)